internal/cli/atlas/onlinearchive/start.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package onlinearchive
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/json"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+type StartOpts struct {
+	cli.GlobalOpts
+	id          string
+	clusterName string
+	store       store.OnlineArchiveUpdater
+}
+
+func (opts *StartOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *StartOpts) Run() error {
+	paused := false
+	archive := &atlas.OnlineArchive{
+		ID:     opts.id,
+		Paused: &paused,
+	}
+	result, err := opts.store.UpdateOnlineArchive(opts.ConfigProjectID(), opts.clusterName, archive)
+
+	if err != nil {
```

Comment:
same [np] w pointy & whitespace

Reply:
we don't have pointy as a dependency in the cli, we could discuss adding it in a separate ticket/PR

Reply:
Do we need to jump through any hoops to add new dependencies?

Reply:
@robcarlan-mlab as long as it's licence is pre approved by legal no, if is a more restrictive license we need clearance from them. Other than that not that I could think of